packages/unmock-core/src/__tests__/generator.test.ts

```diff
@@ -146,9 +146,11 @@ describe("Tests generator", () => {
     });
     expect(resp).toBeDefined();
     if (resp && resp.body !== undefined) {
-      JSON.parse(resp.body).forEach((pet: any) =>
-        expect(pet.id).toEqual("foo"),
-      );
+      if (!(resp.body instanceof Array)) {
+        throw new Error("Response body was not an object but a string");
+      }
+
+      resp.body.forEach((pet: any) => expect(pet.id).toEqual("foo"));
```

Comment:
I would roll this into one expectation. Depending on what you are trying to test, this could be `expect(new Set (resp.body.map(v => v.id))).toEqual(new Set(["foo"]))`

Reply:
Or `expect(resp.body.every((pet: any) => pet.id === "foo")).toBeTruthy();`

Reply:
Note that this line was not added in this PR, but of course can update that as well.

Reply:
True, added originally by me.
The original intent was to be able to debug the different results (i.e. `expected "foo" received "14351"` vs `expected "true" received "false"`). It's a bit of a leftover at this point, would be nice to clean it up.